lib/reservation_generator_helper.rb

```diff
@@ -0,0 +1,96 @@
+module ReservationGeneratorHelper
```

Comment:
Why did you decide to put this into a separate module? I'm not sure I see the benefit, if anything I might prefer to have all of the related methods in a single module, albeit with the bulk of them being `private`. What do you think?